applicationinsights.dev.json

```diff
@@ -28,22 +28,6 @@
             }
           ],
           "percentage": 10
-        },
```

Comment:
same as above

Reply:
Based on discussion with Aled, this is to be removed as per his chat with Peter Philips.